### ISSUE TITLE
profiler\\multiple\\multiple\\multiple.cmd failing on windows arm64

### ISSUE THREAD

**Issue by AndyAyersMS** [dotnet/runtime#98817]
## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=574998
Build error leg or test failing: profiler\\multiple\\multiple\\multiple.cmd
Pull request: https://github.com/dotnet/runtime/pull/98777

## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md#how-to-fill-out-a-known-issue-error-section).

```json
{
  "ErrorMessage": "Unhandled exception. System.Exception: Profiler tests are expected to contain the text 'PROFILER TEST PASSES'",
  "ErrorPattern": "",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```

 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=574998
**Error message validated:** `[Unhandled exception. System.Exception: Profiler tests are expected to contain the text 'PROFILER TEST PASSES'`]
**Result validation:** :white_check_mark: Known issue matched with the provided build.
**Validation performed at:** 2/28/2024 1:41:53 AM UTC

### Report

|Build|Definition|Test|Pull Request|
|---|---|---|---|
|[655183](https://dev.azure.com/dnceng-public/public/_build/results?buildId=655183)|dotnet/runtime|[profiler.multiple.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=655183&view=ms.vss-test-web.build-test-results-tab&runId=16217960&resultId=114231)|dotnet/runtime#101291|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|1|1|1|

**Comment by janvorli** [comment]
I will give it a quick look.

**Comment by tommcdon** [comment]
@davmason is this fixed by #98685?

**Comment by jkotas** [comment]
It does not appear to be fixed. It was just hit by #98741 on windows arm64 checked:

https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-98741-merge-ff52306935e5461bb1/profiler.multiple/1/console.3ed85ab5.log?helixlogtype=result

```
        Requesting detach!!
        RequestProfilerDetach hr=0x0
        Requesting detach!!
        Requesting detach!!
        ProfilerDetachSucceeded _detachCount=1
        RequestProfilerDetach hr=0x0
        RequestProfilerDetach hr=0x0
        ProfilerDetachSucceeded _detachCount=2
        
        Test failed: Profiler tests are expected to contain the text 'PROFILER TEST PASSES' in the console output of the profilee app to indicate a passing test. Usually it is printed from the Shutdown() method of the profiler implementation. This text was not found in the output above.
        Expected: 100
        Actual: -532462766
        END EXECUTION - FAILED
```

**Comment by AndyAyersMS** [comment]
Also failing in jitstress: https://github.com/dotnet/runtime/issues/98787

**Comment by davmason** [comment]
This is a new failure mode, the test waits on an event for 5 minutes and is timing out waiting for all profilers to detach. I made a PR to increase the timeout here #99014

For the jitstress failures, it is marked as [JitOptimizationSensitive](https://github.com/dotnet/runtime/blob/8e59df893cbd831618fffb912b087ad065a394e8/src/tests/profiler/multiple/multiple.csproj#L6), shouldn't it be blocked from running under jitstress?

**Comment by AndyAyersMS** [comment]
I don't know if zapdisable gets auto-detected as jit stress. Technically it's not true jit stress, it just happens to be running as part of the jit stress job.

```
set DOTNET_TieredCompilation=0
set DOTNET_ReadyToRun=0
set DOTNET_ZapDisable=1
```

**Comment by davmason** [comment]
Hopefully fixed with the increased timeout in #99014. If this continues to fail I can disable on arm64

**Comment by jkotas** [comment]
5 minutes is a lot of time. I doubt that increasing it to 10 minutes is going to help.

The most likely explanation of the symptoms is that the test is hitting intermittent hang (deadlock, etc.). Is it possible to modify the test to produce dumps when it is timing out so that we can see what's going on?

**Comment by jkotas** [comment]
Reactivating. This is still failing on number of PRs.

**Comment by kunalspathak** [comment]
Failed again in https://dev.azure.com/dnceng-public/public/_build/results?buildId=655183&view=ms.vss-test-web.build-test-results-tab&runId=16217960&resultId=100465&paneView=debug

**Comment by jkotas** [comment]
The error message can have number of different causes. The root cause of the failure tracked by this issue was fixed. This issue should not be reactivated. A new issue should be opened instead.

**Comment by jkotas** [comment]
>  https://dev.azure.com/dnceng-public/public/_build/results?buildId=655183&view=ms.vss-test-web.build-test-results-tab&runId=16217960&resultId=100465&paneView=debug

This is https://github.com/dotnet/runtime/issues/100114
